data-raw/standard-format-data-prep/flow_data_prep_files/flow_data_prep.Rmd

```diff
@@ -512,14 +512,64 @@ filter(oroville_daily_flows, flow_cfs == 0)
 filter(oroville_daily_flows, is.na(flow_cfs))
 min(oroville_daily_flows$flow_cfs)
 ```
+
+### Lower Feather site
+
+```{r}
+# Find CDEC gage nearby the Lower Feather RST site
+# Use this code to see what data are available
+cdec_datasets("FSB") 
+# For reference - this mapping tool shows CDEC sites: https://cdec.water.ca.gov/webgis/?appid=cdecstation
+
+# Use this code to pull the data from CDEC. 
+boyds_landing_CDEC <- cdec_query(station = "FSB", dur_code = "H", 
```

Comment:
Looks like data doesn't start until 2010 so I might update that query start date